hw_32_form/js/helpers.js

```diff
@@ -0,0 +1,20 @@
+export const Manipulator = {
```

Comment:
Manipulator is a bad name for this project. I know you took it from a previous project. But it is used in games or other projects to key manipulating of something. 
Here you have 3 properties that can't be described with one class, so you can use abstract name `Helpers`

and there is no need to reduce it to one letter when importing.
```javascript
import { Helpers } from "./js/helpers.js";

// ...

let find = Helpers.findinDoc(form);
```